corporate/views/support.py

```diff
@@ -45,7 +46,7 @@ def support_request(request: HttpRequest) -> HttpResponse:
             send_email(
                 "zerver/emails/support_request",
                 to_emails=[FromAddress.SUPPORT],
-                from_name="Zulip Support",
+                from_name=f"{settings.SERVICE_NAME} Support",
```

Comment:
Let's change this to be hardcoded to "Zulip support request", with a comment before this `send_email` call of the form "# Sent to the server's support team, so this email is not user-facing."